docs/plugins/google-cloud.md

```diff
@@ -43,370 +43,8 @@ import { enableGoogleCloudTelemetry } from '@genkit-ai/google-cloud';
 enableGoogleCloudTelemetry();
 ```
 
-When running in production, telemetry will be exported automatically.
+When running in production, telemetry will be exported automatically. See the [Advanced configuration guide](./observability/advanced-configuration.md) for configuration options.
```

Comment:
I think we need some callout re: auth in this doucument itself, since I am not sure it will be synonymous with "configuration".